src/main/java/seedu/duke/Parser.java

```diff
@@ -43,6 +44,8 @@ public Command handleUserCommand(String userInput, ArrayList<University> univers
             // Handle exception for parseInt, use try catch, shift it to another Function thanks
             int indexToRemove = Integer.parseInt(userCommandSecondKeyword);
```

Comment:
This line should put inside the handling part, and handle exception for parseInt for NumberFormatError